core/src/main/scala/com/mesosphere/usi/core/logic/MesosEventsLogic.scala

```diff
@@ -87,17 +87,31 @@ private[core] class MesosEventsLogic(mesosCallFactory: MesosCalls) extends Impli
       intents.withPodRecord(podId, Some(PodRecord(podId, Instant.now(), offer.getAgentId.asModel)))
     }
 
-    val op =
-      newOfferOperation(Mesos.Offer.Operation.Type.LAUNCH, launch = newOfferOperationLaunch(taskInfos.values.flatten))
-
-    val acceptBuilder = MesosCall.Accept
-      .newBuilder()
-      .addOperations(op)
-      .addOfferIds(offer.getId)
-
-    val intents = intentsBuilder.withMesosCall(mesosCallFactory.newAccept(acceptBuilder.build))
+    val offerIntent = if (taskInfos.isEmpty) {
```

Comment:
we renamed the output parameter to `SchedulerEventsBuilder` so I think we should stop using intents also for names of the variables